platform/linux-generic/odp_crypto_ipsecmb.c

```diff
@@ -62,6 +66,10 @@ static const odp_crypto_auth_capability_t auth_capa_zuc_eia3[] = {
 {.digest_len = 4, .key_len = 32, .aad_len = {.min = 0, .max = 0, .inc = 0},
 	.iv_len = 25} };
 
+static const odp_crypto_auth_capability_t auth_capa_snow3g_uia2[] = {
+{.digest_len = 4, .key_len = 16, .aad_len = {.min = 0, .max = 0, .inc = 0},
```

Comment:
The macro for the digest len could be use here.

Reply:
Sure, updated this line with marco.